fido-uaf-core/build.gradle

```diff
@@ -30,6 +30,66 @@ dependencies {
     testCompile 'junit:junit:4.12'
 }
 
+def releaseVersionName = "1.0.7"
```

Comment:
the gradle `release` job reads and bumps the version in `gradle.properties`. Could you use that instead of redefining it here?